modules/cms/classes/Controller.php

```diff
@@ -252,6 +252,13 @@ public function run($url = '/')
             $result = $event;
         }
 
+        /*
+         * Set a default samesite config value for invalid values
+         */
+        if (strtolower(Config::get('session.same_site')) !== 'lax' || 'strict' || 'none') {
```

Comment:
This one doesn't work, should be something like:
```suggestion
        $sameSite = strtolower(Config::get('session.same_site'));
        if ($sameSite !== 'lax' && $sameSite !== 'strict' && $sameSite !== 'none') {
```

Reply:
Maybe better will be to use in_array, `!in_array(strtolower(Config::get('session.same_site')) ['lax', 'strict', 'none'])`

Reply:
Cool cool, thanks for suggestions. Yah going to use `in_array` better performance p.s. you missed an `,` in the middle.